apps/microtvm/zephyr/template_project/src/aot_standalone_demo/main.c

```diff
@@ -229,13 +180,13 @@ void serial_callback(char* message, int len_bytes) {
 }
 
 void main(void) {
+  UARTInit();
+  TVMPlatformInitialize();
   g_cmd_buf_ind = 0;
```

Comment:
initialize the Uart after you are ready to receive stuff

Reply:
UartInit initialize both transmit and receive

Reply:
I guess what @areusch means here (and correct me if I'm wrong) is to init Uart once the platform (including memory,  rpc for example) is initialized. Otherwise, the received data might drop or even cause errors (by trying to write into memory spaces that are not allocated yet)

Reply:
That makes sense. I changed the order.